docs/data/data-grid/list-view/list-view.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Data Grid - List view
+---
+
+# Data Grid - List view [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+<p class="description">Display data in a single-column list view. Can be used to present a more compact grid on smaller screens and mobile devices.</p>
+
+List view can be enabled by providing the `unstable_listView` prop.
+
+Unlike the default grid view, the list view makes no assumptions on how data is presented to end users.
+
+In order to display data in a list view, a `unstable_listColumn` prop must be provided with a `renderCell` function.
+
+:::warning
+This feature is under development and is marked as **unstable**. While you can use the list view feature in production, the API could change in the future.
+:::
+
+{{"demo": "ListView.js", "bg": "inline"}}
+
+## Advanced usage
+
+The list view feature can be combined with [custom subcomponents](/x/react-data-grid/components/) to provide an improved user experience on small screens.
+
+{{"demo": "ListViewAdvanced.js", "bg": "inline", "hideToolbar": true}}
```

Comment:
I wonder if we should switch back to `"iframe": true` for this demo 🤔
These layout jumps do not look great.

https://github.com/user-attachments/assets/e72cc86d-00fd-4ef0-b823-b0eda0d9f789

Reply:
Good spot, seems to be an issue when scrollbars are always visible. Material UI add some padding to account for scrollbars https://github.com/mui/material-ui/blob/e3af9e9850c93d319b2b26f8cd77dca897b4d7f7/packages/material-ui/src/Modal/ModalManager.js#L33.

> I wonder if we should switch back to "iframe": true for this demo 
Agreed, I don't think it's worth all of the hacks to make this demo work.